### PR TITLE
[iOS] Add -convertCoordinate:toPointToView:

### PR DESCRIPTION
**Pull request by alexZhangKai**
My project needs to have the screen locations of points on the map. So I added -convertCoordinate:toPointToView: from mapbox native iOS SDK on react native iOS platform. And I tried to follow the code style as much as possible.

### CODE REVIEW

File: ios/RCTMGL/RCTMGLMapViewManager.m [hunk 1]
```diff
@@ -95,6 +95,28 @@ - (UIView *)view
...
+        id view = viewRegistry[reactTag];
+        
+        if (![view isKindOfClass:[RCTMGLMapView class]]) {
+            RCTLogError(@"Invalid react tag, could not find RCTMGLMapView");
+            return;
+        }
+        
+        RCTMGLMapView *reactMapView = (RCTMGLMapView*)view;
+
+        CGPoint pointInView = [reactMapView convertCoordinate:CLLocationCoordinate2DMake([coordinate[0] doubleValue], [coordinate[1] doubleValue])
```
Comment (nitaliano) [inline ios/RCTMGL/RCTMGLMapViewManager.m]: Would you mind updating coordinates passed in to be in GeoJSON format `[longitude, latitude]`?

**Comment by nitaliano** [comment]
Thanks for the PR, this is something we can merge in and I can follow up with the Android portion in a separate PR(unless that is something you would want to do). I did have one comment above, all of our APIs follow GeoJSON format so if you could update that we can work on getting this merged in

File: ios/RCTMGL/RCTMGLMapViewManager.m (hunk 1 above)
Comment (alexZhangKai) [reply ios/RCTMGL/RCTMGLMapViewManager.m]: Thanks for reviewing. I changed the order in the last commit.

Comment (nitaliano) [reply ios/RCTMGL/RCTMGLMapViewManager.m]: Great, I'll test it out today and then we can merge it in

**Comment by nitaliano** [comment]
@alexZhangKai sorry for the delay I went down the rabbit hole on this PR https://github.com/mapbox/react-native-mapbox-gl/pull/897 I'll have time for getting this merged in and adding the Android portion now

**Pull request merged**